101-create-backup-vault-policy/azuredeploy.json

```diff
@@ -0,0 +1,164 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Vault"
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
```

Comment:
Yes. Its available in 17 regions. Let me add all of them!
https://azure.microsoft.com/en-us/updates/azure-backup-available-in-new-regions/